ui/src/app/views/workspace/component.html

```diff
@@ -177,6 +176,19 @@ <h6>
                      [onClose]="closeShare.bind(this)"
                      [userEmail]="username"
                      [sharing]="sharing"></app-workspace-share>
+<clr-modal [(clrModalOpen)]="googleBucketModal">
+  <h3 class="modal-title">Note</h3>
+  <div class="modal-body">It is All of Us data use policy that researchers should not make copies of or download
+    individual-level data (including taking screenshots or other means of viewing individual-level data) outside of
+    the All of Us research environment without approval from All of Us Resource Access Board (RAB).<br>
+    Notebooks should rarely be downloaded directly from Google Cloud Console, as output cells in Notebooks may
+    contain sensitive individual level data.
+  </div>
+  <div class="modal-footer">`
+    <button type="button" class="btn btn-secondary" (click)="googleBucketModal = false">Cancel</button>
+    <button type="button" class="btn btn-primary" (click)="openGoogleBucket()">Continue</button>
```

Comment:
This should probably be a styled anchor tag so that we get the nice benefits from cmd-click and accessibility benefits.